src/profile/fixefpr.jl

```diff
@@ -102,6 +102,10 @@ function profileβj!(
     ζv = getproperty.(tbl, :ζ)
     βv = getproperty.(tbl, sym)
     val.fwd[sym] = interpolate(βv, ζv, BSplineOrder(4), Natural())
+    if sym == :β1
+        @debug "" sym
+        @debug "" collect(zip(ζv, βv))
```

Comment:
Did you intend to leave this in or is it leftover from debugging?

Reply:
yes and yes. :wink: I was going to leave it for now in case I get the urge to do a deeper dive in the near future about what's causing the reversal. But I can also strip out it and add it if I ever get around to that.

Reply:
I remember seeing an issue quite a while back where having logging macro calls in your code at all came with a performance penalty, even if all uses were in branches that were never executed. I assume it's been fixed in the meantime but don't actually know.

Reply:
Eeek. That's a good reason to drop it though this isn't a particularly hot section of the code.